src/Components/AdditionalRepositoriesTable/__tests__/AdditionalRepositoriesTable.test.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { fireEvent, render, screen } from '@testing-library/react';
+import AdditionalRepositoriesTable from '../AdditionalRepositoriesTable';
+
+jest.mock('../../../hooks/useFeatureFlag');
+import '@testing-library/jest-dom';
+
+const queryClient = new QueryClient();
+
+describe('AdditionalRepositoriesTable', () => {
+  const repositories = [
+    {
+      repositoryLabel: 'label-a',
+    },
+    {
+      repositoryLabel: 'label-b',
+    },
+  ];
+  it('renders correctly', () => {
+    const Table = () => (
+      <QueryClientProvider client={queryClient}>
+        <AdditionalRepositoriesTable repositories={repositories} />
+      </QueryClientProvider>
+    );
+
+    const { container } = render(<Table />);
+
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('when row column headings are clicked', () => {
+  const repositories = [
+    {
+      repositoryLabel: 'label-a',
+    },
+    {
+      repositoryLabel: 'label-b',
+    },
+    {
+      repositoryLabel: 'label-c',
+    },
+  ];
+  it('can sort by label', () => {
+    const Table = () => (
+      <QueryClientProvider client={queryClient}>
+        <AdditionalRepositoriesTable repositories={repositories} />
+      </QueryClientProvider>
+    );
+
+    const { container } = render(<Table />);
+    fireEvent.click(screen.getByText('Label'));
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('can sort by label, reversed', () => {
+    const Table = () => (
+      <QueryClientProvider client={queryClient}>
+        <AdditionalRepositoriesTable repositories={repositories} />
+      </QueryClientProvider>
+    );
+
+    const { container } = render(<Table />);
+    fireEvent.click(screen.getByText('Label'));
+    fireEvent.click(screen.getByText('Label'));
+
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('when using pagination', () => {
+  const repositories = [
+    {
+      repositoryLabel: 'label-a',
+    },
+    {
+      repositoryLabel: 'label-b',
+    },
+    {
+      repositoryLabel: 'label-c',
+    },
+    {
+      repositoryLabel: 'label-d',
+    },
+    {
+      repositoryLabel: 'label-e',
+    },
+    {
+      repositoryLabel: 'label-f',
+    },
+    {
+      repositoryLabel: 'label-g',
+    },
+    {
+      repositoryLabel: 'label-h',
+    },
+    {
+      repositoryLabel: 'label-i',
+    },
+    {
+      repositoryLabel: 'label-j',
+    },
+    {
+      repositoryLabel: 'label-k',
+    },
```

Comment:
We should define this in a less manual way

```suggestion
  const repositories = [...Array(12).keys()].map(id => ({repositoryLabel: `label-${id}`}))
```

This generates 12 repositories with the label "label-id" where id is 0-11